Gemfile

```diff
@@ -205,3 +205,5 @@ gem 'rails-html-sanitizer', '>= 1.0.4'
 
 gem 'mimemagic', '0.4.3'
 gem 'ffi', '>= 1.14.2'
+
+gem 'rubocop', '~> 1.56'
```

Comment:
Why do we need to add this now?

Reply:
it's quite annoying if I want to use rubocop in my local, I need to do `bundle add rubocop` all the time, and then in the push, I need to remove that..

I think it's also useful to be used before creating the PR, so that hound will give feedback much lesser

Reply:
I dont need to use bundle add rubocop and still can run rubocop?

Reply:
Alternatively, do we need rubocop in production? Why not put it under development group?

Reply:
not really sure, maybe need some more configuration on my local machine for this matter

in the meanwhile, I put the rubocop under development group, as suggested